rules/native_binary.bzl

```diff
@@ -20,14 +20,13 @@ do, but they run the wrapped binary directly, instead of through Bash, so they
 don't depend on Bash and work with --shell_exectuable="".
 """
 
-load("//rules/private:copy_file_private.bzl", "copy_bash", "copy_cmd")
```

Comment:
Since we're no longer loading copy_bash/copy_cmd, please remove the dep on //rules/private:copy_file_private in the //rules:native_binary bzl_library

Reply:
Done